### DISCUSSION TITLE
Latest update #320 - Provider Reference

### DISCUSSION THREAD

**Discussion by angelamarie0975** [CMSgov/price-transparency-guide#324] [General]
Can anyone please explain the new schema for the provider reference object? The schema states this is required, but based on the examples, I see that both provider_groups and provider_references is used at different times. I am assuming this means that if we have groups of providers that we can put in the provider_references array, then we can use that or it's the provider_groups array? I am not seeing any explanation for the changes on the site, but maybe I am missing it?

**Comment by shaselton-usds** [comment]
Hi @angelamarie0975 -- Sorry about the confusion and thanks for pointing out the README documentation not reflecting the schema accurately. Does this [PR](https://github.com/CMSgov/price-transparency-guide/pull/327) help out?
